altair/utils/tests/test_schemapi.py

```diff
@@ -245,6 +246,31 @@ def test_copy():
     assert mydct['b'] == dct['b']
     assert mydct['c'] == dct['c']
 
+    # Using copy module
```

Comment:
Please make this a separate test, as it is testing a different copy mechanism (it helps to isolate potential test failures).